plotting/drawhits.py

```diff
@@ -502,6 +530,10 @@ def parse_args():
     processed_events +=1
 # <--- end of the event loop
 
+# Use all fired cells for semiconductor sensors, as quite often there are multiple hits per cellID, with each cellID describing one O(cm^2) large sensor.   
+# This will not be needed anymore once pixel/strip segmentation is added to these detectors in the DD4hep description. Then each pixel has its own cellID.
```

Comment:
This is a useful comment. But I'm wondering if this is the right place for it. Maybe it belongs more to the high level estimations script?